### ISSUE TITLE
problem of 2021/assignment1/two_layer_net

### ISSUE THREAD

**Issue by hustophi** [cs231n/cs231n.github.io#277]
In `two_layer_net.ipynb` , when I use a Solver instance to train a TwoLayerNet, a numerical overflow error raised, which caused the softmax loss `nan`, but I have successfully passed all test above this step. Anyone also encountered this bug? Please give me some advice!
![image](https://user-images.githubusercontent.com/51041892/145682231-fc085a4e-3512-4b76-ba80-78adf026c3a7.png)

**Comment by HumbleHumbert** [comment]
same problem, I have seen many people just change the default learning rate to 1e-3, how did you fix it?
